src/data/markdown/translated-guides/en/02 Using k6/12 Execution context variables.md

```diff
@@ -31,24 +31,30 @@ $ k6 run --vus 10 --iterations 100 script.js
 
 ## \_\_VU and \_\_ITER
 
-**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes
-available to the test script.
+**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes available to the test script.
 
 ### \_\_ITER
 
 A numeric counter with the current iteration number for a specific VU. Zero-based.
 
 ### \_\_VU
 
-Current VU number. The value is assigned incrementally for each new VU instance. One-based.
-However, VU number is 0 while executing the setup and teardown functions.
+Current VU number in use. The value is assigned incrementally for each new VU instance, starting from one. The variable will be 0 while executing the setup and teardown functions.
+
+### ⚠️ To note when running in the k6 Cloud
+
+When you run your tests in the [k6 Cloud](/cloud) you will have additional [environment variables](/using-k6/environment-variables) that will tell you on which server, load zone and distribution of the test you are currently executing. You can read more about them [here](/using-k6/environment-variables). 
+
+The **\_\_VU** environment variable may also behave differently while running a test in the cloud. This is because the **\_\_VU** value is shown _per server instance running you test_. How many servers we use depends on the size of your test and how you spread it across different load zones. So for a larger test, the value of **\_\_VU** will likely not reflect the actual _total_ number of VUs in your test at a given moment.
+
+- For tests using **1-999** VUs per load zone, we use server instances that handle 300 VU's per machine. So for a 900 VU test using one load zone, there will be three instances and the value of **\_\_VU** will vary from 1-300 per machine.
```

Comment:
@Griatch 

I suggest moving all this part out of the k6 OSS guides. 

I think it is better to link to the k6 Cloud docs where we document this part. Otherwise, the information is redundant in both sections and harder to maintain.  I suggest something like:



> You can read [here](https://k6.io/docs/cloud/cloud-faq/general-questions/#how-many-vus-can-be-run-from-the-same-dedicated-ip) how the k6 Cloud allocates VUs per server (aka load generator).

Reply:
Ok, I removed this and linked. I added Mihail's mention of these values 'just being defaults' to the main text though, it seems to make sense there too.